src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
-      opt_callback();
+      opt_callback(true);
```

Comment:
opt_callback(/*sizeChanged*/true); 

Ditto everywhere this is being called

(Check on what the comment format for this is )